src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/Language/RazorLanguageFeatureFlags.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.Razor.Language;
+
+public sealed record class RazorLanguageFeatureFlags(
+    bool ForceRuntimeCodeGeneration)
+{
+    public static RazorLanguageFeatureFlags Default = new(
```

Comment:
I have very mixed feelings about whether there should be a `Default` at all for this. It feels a little presumptuous for something to decide "I don't care about what the language server options think we should do", but at the same time it would be annoying having to pass the feature flags around all the time.

Reply:
Yea. This might be a place where allowing null for the options is okay? And just make sure people pass null explicitly

Reply:
If `RazorLanguageFeatureFlags` will have a default, should the `RazorConfiguration` constructor take `RazorLanguageFeatureFlags? flags = null` and choose `RazorLanguageFeatureFlags.Default` if they're unspecified? That would avoid a lot of the updates in this PR.